maps/away/generic/civilian_station.dm

```diff
@@ -2,7 +2,7 @@
 	name = "civilian station"
 	description = "A modestly-sized independently-owned civilian space station. Many of these exist all throughout inhabited space, offering a place to rest, food to eat, shopping, and refueling - part mall, part motel. This one appears to have been active in the region long before Biesel took control, and an information lookup indicates that it is operated by a small company that is Solarian in origin. This one's transponder says it's open for business!"
 	suffix = "generic/civilian_station.dmm"
-	sectors = list(SECTOR_TAU_CETI, SECTOR_ROMANOVICH, SECTOR_CORP_ZONE, SECTOR_VALLEY_HALE)
+	sectors = list(SECTOR_CORP_ZONE, SECTOR_VALLEY_HALE)
```

Comment:
```suggestion
	sectors = list(SECTOR_TAU_CETI, SECTOR_ROMANOVICH, SECTOR_CORP_ZONE, SECTOR_VALLEY_HALE)
```

Reply:
The civilian station is Solarian in origin and not owned by the SCC, so a different version would be needed for Tau Ceti and the Romanovich. I will potentially make one should we ever end up going back there.